db/migrate/20190813085556_create_api_tokens.rb

```diff
@@ -0,0 +1,8 @@
+class CreateApiTokens < ActiveRecord::Migration[5.2]
+  def change
+    create_table :api_tokens do |t|
+      t.string :token
```

Comment:
value

Reply:
I think, we should name it 'token', cause value is too abstract

Reply:
@rbUUbr token.value? in the context of the API token, it is enough.
imagine your code with token column:
```
@token = token.token if toke.token.present?
```
looks weird